src/endpoint/iam/iam_utils.js

```diff
@@ -82,6 +82,34 @@ function check_iam_path_was_set(iam_path) {
     return iam_path && iam_path !== IAM_DEFAULT_PATH;
 }
 
+/**
+ * parse_max_items converts the input to the needed type
+ * @param {any} input_max_items
+ */
+function parse_max_items(input_max_items) {
+    if (_.isUndefined(input_max_items)) return;
+    const input_type = 'number';
+    const parameter_name = 'MaxItems';
+    if (typeof parameter_name !== input_type) {
```

Comment:
you meant to use typeof input not of the parameter name... but even so, i dont think this type check makes sense as the input will be of type string and will be parsed to integer, so just pass it to Number() and compare to NaN.

Reply:
Yes, I meant, changing, thanks.